x/feemarket/keeper/eip1559_test.go

```diff
@@ -2,107 +2,109 @@ package keeper_test
 			false,
-			func() {
-				// non initial block
-				suite.ctx = suite.ctx.WithBlockHeight(1)
-
-				// Set gas used
-				suite.app.FeeMarketKeeper.SetBlockGasWanted(suite.ctx, 100)
-
-				// Set target/gasLimit through Consensus Param MaxGas
-				blockParams := abci.BlockParams{
-					MaxGas:   100,
-					MaxBytes: 10,
-				}
-				consParams := abci.ConsensusParams{Block: &blockParams}
-				suite.ctx = suite.ctx.WithConsensusParams(&consParams)
-
-				// set ElasticityMultiplier
-				params := suite.app.FeeMarketKeeper.GetParams(suite.ctx)
-				params.ElasticityMultiplier = 1
-				suite.app.FeeMarketKeeper.SetParams(suite.ctx, params)
-			},
+			1,
+			100,
+			sdk.ZeroDec(),
+			suite.app.FeeMarketKeeper.GetParams(suite.ctx).BaseFee.BigInt(),
+		},
+		{
+			"with BaseFee - parent block wanted the same gas as its target, with higher min gas price",
+			false,
+			1,
+			100,
+			sdk.NewDec(1500000000),
 			suite.app.FeeMarketKeeper.GetParams(suite.ctx).BaseFee.BigInt(),
 		},
 		{
 			"with BaseFee - parent block wanted more gas than its target",
 			false,
-			func() {
-				suite.ctx = suite.ctx.WithBlockHeight(1)
-
-				suite.app.FeeMarketKeeper.SetBlockGasWanted(suite.ctx, 200)
-
-				blockParams := abci.BlockParams{
-					MaxGas:   100,
-					MaxBytes: 10,
-				}
-				consParams := abci.ConsensusParams{Block: &blockParams}
-				suite.ctx = suite.ctx.WithConsensusParams(&consParams)
-
-				params := suite.app.FeeMarketKeeper.GetParams(suite.ctx)
-				params.ElasticityMultiplier = 1
-				suite.app.FeeMarketKeeper.SetParams(suite.ctx, params)
-			},
+			1,
+			200,
+			sdk.ZeroDec(),
+			big.NewInt(1125000000),
+		},
+		{
+			"with BaseFee - parent block wanted more gas than its target, with higher min gas price",
+			false,
+			1,
+			200,
+			sdk.NewDec(1500000000),
 			big.NewInt(1125000000),
 		},
 		{
 			"with BaseFee - Parent gas wanted smaller than parent gas target",
 			false,
-			func() {
-				suite.ctx = suite.ctx.WithBlockHeight(1)
-
-				suite.app.FeeMarketKeeper.SetBlockGasWanted(suite.ctx, 50)
-
-				blockParams := abci.BlockParams{
-					MaxGas:   100,
-					MaxBytes: 10,
-				}
-				consParams := abci.ConsensusParams{Block: &blockParams}
-				suite.ctx = suite.ctx.WithConsensusParams(&consParams)
-
-				params := suite.app.FeeMarketKeeper.GetParams(suite.ctx)
-				params.ElasticityMultiplier = 1
-				suite.app.FeeMarketKeeper.SetParams(suite.ctx, params)
-			},
+			1,
+			50,
+			sdk.ZeroDec(),
 			big.NewInt(937500000),
 		},
+		{
+			"with BaseFee - Parent gas wanted smaller than parent gas target, with higher min gas price",
+			false,
+			1,
+			50,
+			sdk.NewDec(1500000000),
+			big.NewInt(1500000000),
+		},
 	}
 	for _, tc := range testCases {
 		suite.Run(fmt.Sprintf("Case %s", tc.name), func() {
 			suite.SetupTest() // reset
+
 			params := suite.app.FeeMarketKeeper.GetParams(suite.ctx)
 			params.NoBaseFee = tc.NoBaseFee
+			params.MinGasPrice = tc.minGasPrice
+			params.ElasticityMultiplier = 1
```

Comment:
why use 1 for the multiplier?

Reply:
I guess this was done for easier readability. I'll change it to the default and make the test titles more descriptive